pagopa/io-app
# chore(IT Wallet): [SIW-1285] Update ITW credential catalog screen

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5934

## Short description
This PR updates the credential catalog in the card onboarding screen, with a check on the ITW trial status.

## List of changes proposed in this pull request
- Added check for trial status in `WalletCardOnboardingScreen` component

## How to test
With the ITW trial status `ACTIVE`, navigate to the wallet home screen and press the "Add to wallet" button in the bottom of the screen. Check that the ITW credential catalog is visible.

## Preview
<img src="https://github.com/user-attachments/assets/713e07bb-6344-4a50-87cc-5c6ba56a4292" width="250" />
